### PR TITLE
Expose Junebug via Nginx

### PR DESCRIPTION
**Pull request by hodgestar**
Currently we expose Junebug directly, which means that web transports that try to expose themselves via Nginx are not exposed at all.

We need to expose Nginx, and then make Junebug available through Nginx.

### CODE REVIEW

**Comment by hodgestar** [comment]
This is now waiting on https://github.com/praekelt/junebug/pull/85 and a new Junebug release. There might be more changes needed after that.

**Comment by hodgestar** [comment]
More changes. Now waiting on https://github.com/praekelt/junebug/issues/86.

**Comment by hodgestar** [comment]
@KaitCrawford @smn Ready for review..

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,14 @@ RUN pip install -q \
...
-    junebug==0.1.1
+    junebug==0.1.3
+
 COPY ./docker/nginx.conf /etc/supervisor/conf.d/nginx.conf
 COPY ./docker/junebug.conf /etc/supervisor/conf.d/junebug.conf
 COPY ./junebug-entrypoint.sh /scripts/
-EXPOSE 8080
+
+RUN rm /etc/nginx/sites-enabled/default
+COPY ./docker/junebug/junebug.nginx /etc/nginx/includes/junebug/junebug.conf
```
Comment (KaitCrawford) [inline Dockerfile:16]: is this url correct? shouldn't it be ./docker/junebug.nginx ? (presuming it's pointing to the file added in this PR)

**Comment by KaitCrawford** [comment]
@hodgestar small query

**Comment by hodgestar** [comment]
@KaitCrawford Tx -- I hadn't pushed the renaming of the file.

**Comment by KaitCrawford** [comment]
:+1:

**Pull request merged**